.dockerignore

```diff
@@ -0,0 +1 @@
+target
```

Comment:
We don't need this right? Shouldn't docker use .gitignore by default?

Reply:
It seems not since it was copied in the context while building the image